migrations/sql/V2021.01.08.15.15__add_previous_now_status.sql

```diff
@@ -0,0 +1,12 @@
+Alter table now_application add column previous_application_status_code character varying(3);
```

Comment:
There should be a foreign key relationship made for this too.

Reply:
There was an issue with the FK - Justin and I opted against it, Can explain after standup and get your opinion?

Reply:
FK has been added.